### PR TITLE
Report button skeleton implemented. It still does nothing since I'm …

### PR DESCRIPTION
**Pull request by amattei777**
…waiting on the back end api though

### CODE REVIEW

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java [hunk 1]
```diff
@@ -63,5 +65,28 @@ private void openPreviewActivity() {
...
+                    public void onClick(DialogInterface dialog, int which) {
+                        //TODO Update the image karma in the database accordingly to the report policy and ad it to the reported pictures
+                    }
+                });
+        builder.setNegativeButton(android.R.string.cancel, new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
+                //If the cancle button is pressed we do nothing
+            }
+        });
```
Comment (JeremyDezalos) [inline app/src/main/java/com/github/wnder/GuessPreviewActivity.java:86]: use lambdas

**Review by JeremyDezalos — CHANGES_REQUESTED** [review]

**Review by JeremyDezalos — APPROVED** [review]

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java (hunk 1 above)
Comment (amattei777) [reply app/src/main/java/com/github/wnder/GuessPreviewActivity.java]: I changed it

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java [hunk 2]
```diff
@@ -15,13 +18,23 @@
     public static final String EXTRA_CAMERALAT = "cameraLat";
     public static final String EXTRA_CAMERALNG = "cameraLng";
 
+    private Button guessButton;
+    private Button skipButton;
+    private Button reportButton;
+
```
Comment (leonardvaney) [inline app/src/main/java/com/github/wnder/GuessPreviewActivity.java:24]: Is it necessary to keep it as a variable if the button in itself will not change (except maybe the listener associated)?

**Review by leonardvaney — APPROVED** [review]

File: app/src/main/java/com/github/wnder/GuessPreviewActivity.java (hunk 2 above)
Comment (amattei777) [reply app/src/main/java/com/github/wnder/GuessPreviewActivity.java]: It is true that for now those private variables are useless.
I removed them.

**Pull request merged**